data.sql

```diff
@@ -13,3 +13,43 @@ VALUES
   ('Gabumon', '2018-11-15', 2, TRUE, 8, 'unknown'),
   ('Pikachu', '2021-01-07', 1, FALSE, 15.04, 'unknown'),
   ('Devimon', '2017-05-12', 5, TRUE, 11, 'unknown');
+
+
+-- data for the "owners" table
+INSERT INTO owners (full_name, age) VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+-- data for the "species" table
+INSERT INTO species (name) VALUES
+('Pokemon'),
+('Digimon');
+
+-- Sam Smith owns Agumon:
```

Comment:
- - Screenshot reference:
![Selection_027](https://github.com/DevKaranJ/vet-clinic/assets/107632780/18d20c78-1b54-4f28-9265-38b530060ffe)

- [ ] You missed inserting this part of the requirement into your table, According to the requirement, you need to modify your inserted animal to include `species_id value` as I have pointed out in the screenshot above. I suggest you go through the requirements and update this, and do well to rerun the queries and update your screenshots. :smile: :wave: